inventories/sirius/group_vars/all

```diff
@@ -95,20 +95,73 @@ pkg_version_bpm_scripts: "master"
 pkg_version_sirius_epics_screens: "master"
 
-pkg_version_hlacon_conda_qt: "5.12.9"
-pkg_version_hlacon_conda_pyqt: "5.12.3"
-pkg_version_hlacon_conda_pydm: "1.10.4"
-pkg_version_hlacon_conda_siriushlacon: "2.4.1"
-
 # === fac docker services ===
 
 pkg_version_docker_machine_applications: "v1.1.0"
 pkg_version_docker_ctrl_sys_consts: "v1.0.0"
 
-# === fac conda environment ===
-fac_conda_prefix: /opt/conda/envs/sirius
+# === conda environments ===
 
 conda_environments:
+  - name: GAS Conda Environment
+
+    conda_prefix: /opt/conda/envs/gas
+    conda_python_version: '3.9.2'
+
+    conda_packages:
+      - qt=5.12.9
+      - pyqt=5.12.3
+      - pydm=1.10.4
+
+    conda_pip_packages:
+      - siriushlacon==2.4.1
+
+    conda_activate_templates: []
+
+    conda_deactivate_templates: []
+
+    conda_clone_install_make: []
+
+    conda_app_executables:
+      - name:          sirius-hla-as-ap-bbb-monitor
+        executable:    sirius-hla-as-ap-bbb-monitor
+      - name:          sirius-hla-as-ap-conlauncher
+        executable:    sirius-hla-as-ap-conlauncher
+      - name:          sirius-hla-as-ap-countingpru
+        executable:    sirius-hla-as-ap-countingpru
+      - name:          sirius-hla-as-ap-mbtemp
+        executable:    sirius-hla-as-ap-mbtemp
+      - name:          sirius-hla-as-ap-pctrl
+        executable:    sirius-hla-as-ap-pctrl
+      - name:          sirius-hla-as-ps-regatron
+        executable:    sirius-hla-as-ps-regatron
+      - name:          sirius-hla-as-ps-regatron-individual
+        executable:    sirius-hla-as-ps-regatron-individual
+      - name:          sirius-hla-as-pu-spixconv
+        executable:    sirius-hla-as-pu-spixconv
+      - name:          sirius-hla-as-va-agilent4uhv
+        executable:    sirius-hla-as-va-agilent4uhv
+      - name:          sirius-hla-as-va-agilent4uhv-device
+        executable:    sirius-hla-as-va-agilent4uhv-device
+      - name:          sirius-hla-as-va-mks937b
+        executable:    sirius-hla-as-va-mks937b
+      - name:          sirius-hla-bo-va-agilent4uhv-overview
+        executable:    sirius-hla-bo-va-agilent4uhv-overview
+      - name:          sirius-hla-bo-va-mks937b-overview
+        executable:    sirius-hla-bo-va-mks937b-overview
+      - name:          sirius-hla-si-va-agilent4uhv-overview
+        executable:    sirius-hla-si-va-agilent4uhv-overview
+      - name:          sirius-hla-si-va-mks937b-overview
+        executable:    sirius-hla-si-va-mks937b-overview
+      - name:          sirius-hla-tb-va-agilent4uhv-overview
+        executable:    sirius-hla-tb-va-agilent4uhv-overview
+      - name:          sirius-hla-tb-va-mks937b-overview
+        executable:    sirius-hla-tb-va-mks937b-overview
+      - name:          sirius-hla-ts-va-agilent4uhv-overview
+        executable:    sirius-hla-ts-va-agilent4uhv-overview
+      - name:          sirius-hla-ts-va-mks937b-overview
+        executable:    sirius-hla-ts-va-mks937b-overview
+
```

Comment:
@anacso17 @edupcoelho I've moved the environment setup from the role hlacon into the `conda_environments` list

Reply:
wouldn't the executables be the `.py` scripts?